src/Styling/SubMenu.css

```diff
@@ -29,12 +24,83 @@
     align-items: center;
     text-decoration: none;
     color: #f5f5f5;
-    font-size: 18px;
 }
 /* TODO; find the difference between background color and background and when to use it */
```

Comment:
good use of TODO comment, but it’s best to address TODO’s asap to avoid technical debt within your code. I would recommend doing this research and applying any changes(if needed) within your next PR.

Reply:
there's no difference in the result, since [background] actually is a shorthand for

background-color  
background-image  
background-position  
background-repeat  
background-attachment  
background-clip  
background-origin  
background-size
 besides the [background-color] using the background shorthand you could also add one or more values without repeating any other background-* property more than once.

Reply:
Good job on the research! Now you can delete the TODO you commented on line 29 and make any changes in your code(if needed).